lib/src/api/engine/any/native.rs

```diff
@@ -174,14 +174,14 @@ impl Connection for Any {
 						let maybe_connector = address.config.tls_config.map(Connector::from);
 						#[cfg(not(any(feature = "native-tls", feature = "rustls")))]
 						let maybe_connector = None;
+						#[allow(deprecated)]
 						let config = WebSocketConfig {
-							max_send_queue: match capacity {
-								0 => None,
-								capacity => Some(capacity),
-							},
+							max_send_queue: None,
```

Comment:
We should remove this line entirely so that we don't have to `#[allow(deprecated)]` otherwise the compiler won't notify us of future deprecations for that struct's fields.